### ISSUE TITLE
Position and re-size gauges on the video

### ISSUE THREAD

**Issue by peregin** [peregin/gps-overlay-on-video#7]
Selecteable gauges.
Position and re-size on the fly.

**Comment by greglondon** [comment]
Hi, Can't get drag'n'drop or re-size the gauges. Do I do anything wrong or is this not implemented yet?

**Comment by peregin** [comment]
hi @greglondon , unfortunately it is not yet implemented

**Comment by Marlow925** [comment]
Unfortunately the lack of being able to remove default gauges from the video overlay or adding new ones is a real hamper here.

This could also be used for motorsport, but for that it would require, that for example the speed gauge could be placed with a higher max speed. Right now, it seems that I can't remove any of the overlay elements, adjust the parameters nor move them to a different position.

**Comment by peregin** [comment]
hi @Marlow925 , unfortunately is not yet implemented. I will try to do it over the holidays.

**Comment by Marlow925** [comment]
> hi @Marlow925 , unfortunately is not yet implemented. I will try to do it over the holidays.

In the interim ... could we get a "motorsport" or "car" profile, that only has the speedo with a max speed of maybe 250 km/h ?

As a short term stop gap ?

**Comment by peregin** [comment]
> > hi @Marlow925 , unfortunately is not yet implemented. I will try to do it over the holidays.
> 
> In the interim ... could we get a "motorsport" or "motorbike" or "car" template, that only has the speedo with a max speed of maybe 250 km/h ?
> 
> As a short term stop gap ?

Good idea, basically would do #9 , the predefined templates, let's do that, give me a few days.
